packages/compass-e2e-tests/helpers/commands/exists-eventually.ts

```diff
@@ -3,7 +3,7 @@ import type { CompassBrowser } from '../compass-browser';
 export async function existsEventually(
   browser: CompassBrowser,
   selector: string,
-  timeout = 10000
+  timeout = 1000
```

Comment:
This seems to be faster than before, but best to keep an eye on it. It is unfortunate that we have to wait to see if a modal appears. Although technically we now know if we expect it or not, but that's still a bit hacky.

Point is that this is time added to every test because it has to reach this timeout to see if there are modals to interact with.

As things are written now, anyway. Could change things down the line.

Reply:
Ugh. 1 second is not long enough in CI for when it does show up, so I'll have to rethink this now.